packages/graphiql/src/new-components/List/List.js

```diff
@@ -1,44 +0,0 @@
-/** @jsx jsx */
```

Comment:
lets make sure the file indexes are honored, this one should have been renamed as well

Reply:
@acao Likewise renamed

Reply:
this file still shows as deleted, when it's been moved and changed slightly. did you use `git add` when renaming files for this PR? it's tricky if you don't use `git add`

Reply:
I did use git add. However was having some difficulty with the precommit hooks. The first time I tried, it failed and the cli output suggested I call a particular git stash command. I did so and then that deletion appeared (as did some merge conflicts I had to manually resolce). 

I later realised that all I needed to do to remedy failing precommit hooks was to revert unstaged changes.